docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-creating-dashboard-proc.adoc

```diff
@@ -0,0 +1,9 @@
+[id='building-custom-dashboard-widgets-creating-dashboard-proc']
+= Creating a custom dashboard widget
+
+There are four stages in the creation of a custom dashboard widget:
```

Comment:
In order to create custom dashboards users must combine some of the workbench available features (data sets, pages, navigation trees and permissions) to create and publish them. Dashboards are nothing but pages  containing at least one reporting component. 

There are four stages in the creation of a custom dashboard:

Reply:
suggestion accepted.
Changed the sentences to make them grammatically correct and conform to red hat guidelines.